prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt

```diff
@@ -203,21 +203,26 @@ object CustomFHIRFunctions : FhirPathFunctions {
         val date = focus.first()
```

Comment:
Probably ought to have some description of the order and type of parameters in the function comment.